tests/languages/naniscript/label_feature.test

```diff
@@ -0,0 +1,24 @@
+#
+# Section1
+#Section2
+# Section4
+#			SectionWithMultipleTabsBefore
+## Section3
+# Section with multiple words
+	# Section with
+	# Section with tab

```

Comment:
Are these valid sections or not? The JSON says no but the name of those "sections" makes it seems like they should be valid.

Reply:
Those are not valid. Not sure which JSON you're referring to.

Reply:
> Those are not valid.

Ok. It's all good then.

> Not sure which JSON you're referring to.

The JSON of the expected token stream below.